app/views/clients/_demographic_row.haml

```diff
@@ -1,5 +1,5 @@
 - pii = client.pii_provider(user: current_user)
-%tr
+%tr{'data-test-client-id' => client.id}
```

Comment:
Was this for testing?  If not, maybe we should remove `test` from the name? 
I think including the source client id in the view is a great idea.